test/manual/js/need.js

```diff
@@ -106,5 +107,6 @@ setTimeout(function() {
                 : e.panel.insertBefore(createPanelElement(e.index - 1));
         }
     });
+    console.log("APPENDING");
```

Comment:
Is it need?

Reply:
I think that's just some leftover I made while testing.
I'll remove it on later commit.